helm/postgres/values.yaml

```diff
@@ -5,6 +5,21 @@ dbName: plural
 ownerChart: null
 suffix: ""
 infix: "-postgres"
+# Example extra user definition:
+#
+# extraUsers:
+# - user1:
+#   - nologin
+# - user2: {}
+#
+# Valid user properties can be found here:
+# https://github.com/zalando/postgres-operator/blob/c895e8f61fc394602bc3151c5b8f00954dcf5d8e/manifests/postgresql.crd.yaml#L559-L587
+extraUsers: []
+# Example extra db definition:
+#
+# extraDBs:
+#   testdb1: user1
```

Comment:
I think it's actually `user: owner`, but doesn't matter much.

Reply:
Yep, it is. It's mapped as `{{ $value }}: {{ $key }}`. I just thought it would be more natural to define it here as (db: owner)